students/luke/s05/mailroom.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python3
+
+"""
+https://uwpce-pythoncert.github.io/PythonCertDevel/exercises/mailroom.html#exercise-mailroom
+"""
+
+# Store donors/history in dictionary
+donor_list = {"alice": [5],
+              "bob": [1, 2],
+              "carol": [1, 5, 100],
+              "dan": [5],
+              "erin": [2, 1]}
+
+# Format of thank-you notes
+THANK_YOU = "Thank you, {}, for your donation(s) of ${}!\n"
+
+# Destination file
+OUTFILE = "{}.txt"
```

Comment:
Cool, it never occurred to me you could put in {} before sticking the format method after it, but makes total sense.